gdk/PixbufLoader.custom

```diff
@@ -47,14 +49,18 @@
 			return this.Write (bytes, (ulong) count);
 		}
 
-		private void LoadFromStream (System.IO.Stream input)
-		{
-			byte [] buffer = new byte [8192];
-			int n;
+        const int bufSize = 8192;
+        static ByteArrayPool pool = new ByteArrayPool ();
```

Comment:
Then we have a problem with API additions across versions. We wouldn't be able to use it in XS unless we require gtk# 2.12.X installed.